miner/src/client.rs

```diff
@@ -149,10 +149,6 @@ impl Client {
                 }
                 Err(e) => {
                     error!("rpc call submit_block error: {:?}", e);
-                    sentry::capture_message(
```

Comment:
Is this needed for this PR?

Reply:
Yes, it can reduce noises in Sentry.

Reply:
Not a part of Windows support, just port together.

Reply:
Changed PR title